metrics/run/collect_metrics.go

```diff
@@ -138,89 +143,199 @@ func main() {
 	wg.Wait()
 
 	log.Println("Computed metrics")
-	log.Printf("Writing metrics to Google Cloud Storage bucket: %s\n",
-		*outputGcsBucket)
+	log.Println("Uploading metrics")
 
 	outputBucket := gcsClient.Bucket(*outputGcsBucket)
-	outputCtx := storage.Context{
-		Context: ctx,
-		Client:  gcsClient,
-		Bucket: &storage.Bucket{
-			Name:   outputGcsBucket,
-			Handle: outputBucket,
+	datastoreClient, err := datastore.NewClient(ctx, *projectId)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Surprised we're not returning here - won't datastoreClient be nil or unusable in case of error?

Reply:
If I understand correctly, `log.Fatal*()` functions terminate the process. Do you think it's clearer to also return?

Reply:
> Fatal is equivalent to Print() followed by a call to os.Exit(1).

TIL.